app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1707,6 +1792,22 @@ class VaultAddEditViewModel @Inject constructor(
         getRequestAndRegisterCredential()
     }
 
+    private fun handleSshKeyCipherItemsFeatureFlagReceive(
+        action: VaultAddEditAction.Internal.SshKeyCipherItemsFeatureFlagReceive,
+    ) {
+        mutableStateFlow.update {
+            it.copy(
+                supportedItemTypes = if (action.enabled) {
```

Comment:
maybe we could toss this in a private function: `private fun getSupportedItemTypes(): List<ItemTypeOption>` since we are using it twice.